packages/app/src/index.js

```diff
@@ -8,19 +8,31 @@ import Home from './components/pages/Home/index';
 import SearchResults from './components/SearchResultsCard';
 import { NominationsDataProvider } from './utils/context/NominationsContext';
 import { SearchResultDataProvider } from './utils/context/SearchResultsContext';
+import { ActiveNominationProvider } from './utils/context/ActiveNominationContext';
+import NominationsPage from './components/pages/NominationPage';
 import './App.css';
 
-
 ReactDOM.render(
   <Router>
     <React.StrictMode>
       <Switch>
         <NominationsDataProvider>
           <SearchResultDataProvider>
-            <Route exact path={'/'} component={Home} />
-            <Route exact path={'/login'} component={Login} />
-            <Route exact path={'/searchresults'} component={SearchResults} />
-            <Route path="/nomination/:id" exact component={NominationShow} />
+            <ActiveNominationProvider>
+              <Route path={['/', '/home']} component={Home} />
+              <Route exact path={'/login'} component={Login} />
+              <Route exact path={'/searchresults'} component={SearchResults} />
+              <Route path="/nomination/:id" exact component={NominationsPage} />
+              <Route path="/applicationdetails">
+                <div
+                  style={{
+                    margin: '0 auto',
+                    backgroundColor: 'grey',
+                    padding: '2em',
+                  }}
+                ></div>
```

Comment:
What's the reason for this div with the inline styles here?

Reply:
I messed up the merge conflict will resolve with updates